scripts/setupPythonEnvironment.bash

```diff
@@ -84,13 +76,47 @@ then
 fi
 
 
+# Check for a predefined package directory
+echo "Checking for python packages..."
+if [[ -z "${PACKAGE_DIR}" ]]
+then
+    echo "Cloning the GEOS python package repository..."
+    TMP_CLONE_DIR=$(mktemp -d)
+    PACKAGE_DIR=$TMP_CLONE_DIR/geosPythonPackages
+    git clone --depth 1 --branch main --single-branch https://github.com/GEOS-DEV/geosPythonPackages.git $PACKAGE_DIR
+elif [ ! -d "${PACKAGE_DIR}/geosx_xml_tools_package" ]
```

Comment:
This is just a quick check to see if the directory is close to correct.  There are later error messages that will be printed if other packages are broken/missing.